### ISSUE TITLE
Window Leaked on Play Services not installed

### ISSUE THREAD

**Issue by leonardo2204** [firebase/FirebaseUI-Android#471]
When running Firebase-UI on my emulator, I get a blank screen and a Leak error.

 * Android device: Emulator Nexus 5x API 22 with Play Services
  * Android OS version: 5.1
  * Google Play Services version:  9.8.77
  * Firebase/Play Services SDK version: 10.0.0
  * FirebaseUI version: 1.0.1

#### Observed Results:

Warning: Google Play services out of date.  Requires 10084000 but found 9877280
Stack of the error:
```Java
Activity com.firebase.ui.auth.KickoffActivity has leaked window com.android.internal.policy.impl.PhoneWindow$DecorView{15693dd7 V.E..... R....... 0,0-1026,515} that was originally added here
at android.view.ViewRootImpl.<init>(ViewRootImpl.java:363)
at android.view.WindowManagerGlobal.addView(WindowManagerGlobal.java:271)
at android.view.WindowManagerImpl.addView(WindowManagerImpl.java:85)
at android.app.Dialog.show(Dialog.java:298)
at com.firebase.ui.auth.util.PlayServicesHelper.makePlayServicesAvailable(PlayServicesHelper.java:93)
at com.firebase.ui.auth.util.signincontainer.SignInDelegate.onCreate(SignInDelegate.java:92)
at android.support.v4.app.Fragment.performCreate(Fragment.java:2075)
at android.support.v4.app.FragmentManagerImpl.moveToState(FragmentManager.java:1060)
at android.support.v4.app.FragmentManagerImpl.moveToState(FragmentManager.java:1295)
at android.support.v4.app.BackStackRecord.run(BackStackRecord.java:801)
at android.support.v4.app.FragmentManagerImpl.execPendingActions(FragmentManager.java:1682)
at android.support.v4.app.FragmentController.execPendingActions(FragmentController.java:388)
at android.support.v4.app.FragmentActivity.onStart(FragmentActivity.java:607)
at android.support.v7.app.AppCompatActivity.onStart(AppCompatActivity.java:181)
at android.app.Instrumentation.callActivityOnStart(Instrumentation.java:1236)
at android.app.Activity.performStart(Activity.java:6006)
at android.app.ActivityThread.performLaunchActivity(ActivityThread.java:2288)
at android.app.ActivityThread.handleLaunchActivity(ActivityThread.java:2387)
at android.app.ActivityThread.access$800(ActivityThread.java:151)
at android.app.ActivityThread$H.handleMessage(ActivityThread.java:1303)
at android.os.Handler.dispatchMessage(Handler.java:102)
at android.os.Looper.loop(Looper.java:135)
at android.app.ActivityThread.main(ActivityThread.java:5254)
at java.lang.reflect.Method.invoke(Native Method)
at java.lang.reflect.Method.invoke(Method.java:372)
at com.android.internal.os.ZygoteInit$MethodAndArgsCaller.run(ZygoteInit.java:903)
at com.android.internal.os.ZygoteInit.main(ZygoteInit.java:698)
```
#### Relevant Code:
MainActivity
  ```Java
onCreate(){
startActivityForResult(AuthUI.getInstance().createSignInIntentBuilder()
            .setProviders(Arrays.asList(
                    new AuthUI.IdpConfig.Builder(AuthUI.EMAIL_PROVIDER).build(),
                    new AuthUI.IdpConfig.Builder(AuthUI.GOOGLE_PROVIDER).build(),
                    new AuthUI.IdpConfig.Builder(AuthUI.FACEBOOK_PROVIDER).build()))
                    .setIsSmartLockEnabled(false)
                    .build(), RC_SIGN_IN);
}
  ```

**Comment by SUPERCILEX** [comment]
@leonardo2204 Haha! @samtstern just merged a PR a few hours ago fixing this! 😄 Anyway, thanks for the bug report, the fix will be available in FirebaseUI v1.1.0 coming in January.

**Comment by leonardo2204** [comment]
@SUPERCILEX hahaha cool, I'll wait then !
Thank you

**Comment by azakhary** [comment]
Is there any nightly build or something where this fix can be found till January?

**Comment by samtstern** [comment]
@azakhary you can build from the `version-1.1.0-dev` branch and use that if you want, but it has not been fully QA'd yet!

**Comment by SUPERCILEX** [comment]
@azakhary Like @samtstern said, you can build from the dev branch, but the fragmented posdcast mentioned this amazing service a while back that makes life much easier doing so: [jitpack.io](https://www.jitpack.io). I [use it](https://github.com/SUPERCILEX/FirebaseUI-Android/#installation) to test out my PRs and use unmerged ones like #309. For you, I think the safest way to use a develop version of FirebaseUI would be to target a single commit like so: compile `compile 'com.github.firebase.FirebaseUI-Android:firebase-ui-auth:63e3f9517a9cd87e4d3c96b4370cf85954334bad'`

PS: look at the end of that commit hash (it has the word "bad") so maybe this a bad omen 😄.

Anyway, I hope this helped you out!

**Comment by samtstern** [comment]
A fix for this issue has been released in version `1.1.0`
